include/ajax.thread.php

```diff
@@ -108,7 +108,7 @@ function addCollaborator($tid, $type=null, $uid=0) {
                                 $vars, $errors))) {
                     $info = array('msg' => sprintf(__('%s added as a collaborator'),
                                 Format::htmlchars($c->getName())));
-                    $c->setCc($c->active);
+                    $c->setCc();
```

Comment:
Wouldn't this change make `$c->active = false;` unnecessary (set in `class.thread.php` line 190).

Reply:
Well instead of passing the property I was leaving it empty because 1.) $active defaults to true in setCc and 2.) we are reverting this feature for agent added collabs so I removed it altogether so it always enables the collabs that agents add via the modal. With this being said I have the $thisstaff check in Thread::addCollab so I can pass it without worries because the property will always be set to true when agents are adding collabs. I can update this momentarily.

Reply:
Updated: Leaving `$c->active` as this will always be set to `true` for Agent added collabs